dockers/docker-platform-monitor/docker_init.j2

```diff
@@ -88,6 +89,15 @@ fi
 if [ -e $SENSORS_CONF_FILE ]; then
     HAVE_SENSORS_CONF=1
     mkdir -p /etc/sensors.d
+    PSU_SENSORS_CONF_UPDATER="/usr/share/sonic/platform/psu_sensors_conf_updater"
+    if [ -e $PSU_SENSORS_CONF_UPDATER ]; then
+        source $PSU_SENSORS_CONF_UPDATER 
+        update_psu_sensors_configuration $SENSORS_CONF_FILE
+        if [ -f /tmp/sensors.conf]; then
+            HAVE_PSU_SENSORS_DYNAMIC_CONF=1
```

Comment:
didn't see how "HAVE_PSU_SENSORS_DYNAMIC_CONF" will be used, missing code or we can remove it?

Reply:
removed.